apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -42,6 +42,8 @@ import {
   Courses,
   Subjects
 } from '@cdo/apps/generated/pd/sharedWorkshopConstants';
+import HelpTip from '../../../../lib/ui/HelpTip';
+import experiments from '../../../../util/experiments';
```

Comment:
Is it shorter to use an absolute reference?

Reply:
Hah! Probably.  I'm a fan of relative references because my editor doesn't follow our `@cdo/apps` alias, but you're right, we should optimize for readability here.  I'll update this.